openedx/core/djangoapps/programs/utils.py

```diff
@@ -351,4 +351,12 @@ def supplement_program_data(program_data, user):
             # TODO: Currently unavailable on LMS.
             run_mode['marketing_url'] = ''
 
+            certificate_data = certificate_api.certificate_downloadable_status(user, course_key)
+            certificate_uuid = certificate_data.get('uuid')
+            if certificate_uuid:
```

Comment:
The default for new courses is to use HTML certificates, yes, and the UUID endpoint is the preferred route.  So, I'm okay keeping this as-is, but thought it was worth a brief discussion.